parm/fv3lam_rrfs.xml

```diff
@@ -1923,6 +1985,54 @@
            <scale>5.0</scale>
        </param>
 
+       <param>
+           <shortname>CRITICAL_ANGLE</shortname>
+           <pname>CANGLE</pname>
+           <scale>5.0</scale>
+       </param>
+
+       <param>
+           <shortname>DENDRITIC_LAYER_DEPTH</shortname>
+           <pname>LAYTH</pname>
+           <table_info>NCEP</table_info>
+           <fixed_sfc1_type>isothermal</fixed_sfc1_type>
+           <level>261.</level>
+           <fixed_sfc2_type>isothermal</fixed_sfc2_type>
+           <level2>256.</level2>
+           <scale>4.0</scale>
+       </param>
+
+       <param>
+           <shortname>ENHANCED_STRETCHING_POTENTIAL</shortname>
+           <pname>ESP</pname>
```

Comment:
@AndrewBenjamin-NOAA This field was not generated in your test. You may remove it.